docs/_pages/collections.md

```diff
@@ -190,6 +190,23 @@ collection.Should().SatisfyRespectively(
     });
 ```
 
+If you need to perform the same assertion on all elements of a collection:
+
+```csharp
+var collection = new []
+{
+    new { Id = 1, Name = "John", Attributes = new string[] { } },
+    new { Id = 2, Name = "Jane", Attributes = new string[] { "attr" } }
+};
+collection.Should().AllSatisfy(
```

Comment:
🙃 I'm nitpicking, but the layout is a bit weird. Normally we keep the `x =>` on the same line as `AllSatisfy`.